templates/layouts/_base.html

```diff
@@ -85,7 +85,7 @@
       (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
       new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
       j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtm.js?id='+i+dl+ '&gtm_auth={{ google_tag_manager_auth }}&gtm_preview={{ google_tag_manager_preview }}&gtm_cookies_win=x';f.parentNode.insertBefore(j,f);
+      'https://www.googletagmanager.com/gtm.js?id='+i+dl+ '&gtm_auth={{ google_tag_manager_auth }}&gtm_cookies_win=x';f.parentNode.insertBefore(j,f);
```

Comment:
So if `google_tag_manager_preview` is no longer required then it should be removed from runner and the pipelines.

Reply:
Not yet. It needs to be tested across the different environments. I have separate branches for that. Once it's been confirmed its working in multiple integrated environments that's when it will be removed.

Reply:
I thought this was tested in staging?.

Reply:
It has been. However, there are different GTM containers for different environments. Testing only in staging (now we've removed the env value) does not confirm that the newly formed request sends the events to the correct GTM container. That can only be confirmed by testing in more environments (essentially WL/BL) hence the need to roll this change out to other integrated environments.

Reply:
Can we make sure we have a card to remove this from runner and pipelines if other envs works as expected then.